docker-compose.yml

```diff
@@ -1,4 +1,37 @@
 services:
+  zookeeper:
+    image: zookeeper:3.8
+    volumes:
+      - zkdata:/data
+      - zkdatalog:/datalog
+    ports:
+      - 2181:2181
+
+  kafka:
+    image: wurstmeister/kafka:2.13-2.8.1 # This tag is <scala version>-<kafka version>
+    volumes:
+      - kafkadata:/kafka/kafka-logs-kafka
+    ports:
+      - 9092:9092
+    hostname: kafka
+    depends_on:
+      - zookeeper
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://:29092,OUTSIDE://localhost:9092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT
+      KAFKA_LISTENERS: INSIDE://:29092,OUTSIDE://:9092
+      KAFKA_INTER_BROKER_LISTENER_NAME: INSIDE
+
+  redpanda:
+    image: docker.redpanda.com/vectorized/console:master-cb51d72
+    depends_on:
+      - kafka
+      - eventsapi
```

Comment:
Oh, sorry, my previous comment was not to add `eventsapi` here (since `redpanda` does not depend on `eventsapi`), but instead add the `depends_on` for `kafka` to the `eventsapi` service so it looks something like:

```yaml
  eventsapi:
    build:
      context: .
    depends_on:
      - kafka
```

Reply:
That sounds fine to me / the reason for the error is understandable. I am however, confused when you saw this error, I'm not reproducing it. You said as soon as you bring the environment up with `docker compose up -d`?

Reply:
It is a race condition so it may not always happen. If you do the following a few times you might see it after one of the `up`s, though:

```bash
docker-compose down
docker-compose up -d
```